docs/assets/2.3-2.4-cleanup-orphaned-svcat-resources.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+echo "Deleting deprecated Application Connector resources"
+
+kubectl -n kyma-integration delete deployment application-broker
+kubectl -n kyma-integration delete service application-broker
+kubectl -n kyma-integration delete configmap app-broker-config-map
+kubectl -n kyma-integration delete authorizationpolicy application-connector-application-broker
+kubectl delete clusterrolebinding application-broker
+kubectl delete clusterrole application-broker
+kubectl -n kyma-integration delete serviceaccount application-broker
+
+kubectl -n kyma-integration delete statefulset application-operator
+# Make sure that Application Operator is removed before triggering the Service Instances deletion
+kubectl -n kyma-integration wait --for delete pod --selector=control-plane=application-operator
+kubectl -n kyma-integration delete service application-operator-health
+kubectl -n kyma-integration delete service application-operator-service
+kubectl -n kyma-integration delete virtualservice application-operator
+kubectl -n kyma-integration delete destinationrule application-operator-health-rule
+kubectl delete podsecuritypolicy application-operator
+kubectl delete clusterrolebinding application-operator
+kubectl delete clusterrole application-operator
+kubectl -n kyma-integration delete serviceaccount application-operator
+
+kubectl delete serviceinstance --all-namespaces --all
```

Comment:
Won't it affect the BTP ServiceInstances as well? The name is the same, but the group differs.

Anyways, the `serviceinstances.servicecatalog.k8s.io` CRD is removed at the end of the script, will it remove the CRs as well? If yes - we can skip that part, I am not sure we should directly influence what customer provisioned in their cluster.

Reply:
Hmm, you may be right. I'll delete this command as it's not a part of the Application Connector anyway